pkg/publish.go

```diff
@@ -77,16 +82,38 @@ func Publish(file string, rootID string, dryRun bool) (id string, err error) {
 			return "", err
 		}
 
+		imageSrcFiles := GetImageSrcFiles(c.Body.Storage.Value, file)
+		if len(imageSrcFiles) != 0 {
+			c.Body.Storage.Value, err = ReplaceImagesWithAttachments(imageSrcFiles, file, c.Body.Storage.Value, c.ID, wiki, uri)
+			if err != nil {
+				return "", err
+			}
+
+			c.Version.Number += 1
+			c, err = wiki.UpdateContent(c)
+			if err != nil {
+				return "", err
+			}
+		}
+
 	} else {
+		newPageContent := src.Output()
```

Comment:
This seems to be more aptly named `sourceOutput`, and could be defined closer to where it is used.

Reply:
- [ ] change variable name